vendor/ansible_collections/nvidia/bare_metal/plugins/inventory/bmm.py

```
@@ -0,0 +1,477 @@
+            self._add_host_to_keyed_groups(
+                self.get_option('keyed_groups'), instance, name, strict=strict,
+            )
+
+    def _populate_topology(self, client, group_prefix):
+        """Query sites, VPCs, instance types, allocations, and IB partitions."""
+
+        # --- Tenant (needed as a filter for downstream queries) ---
+        tenant_path = '/v2/org/{org}/carbide/tenant/current'
+        try:
+            tenant = client.get(tenant_path)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)